tests/virt_control_plane_test.go

```diff
@@ -45,9 +45,6 @@ const (
 
 var _ = Describe("KubeVirt control plane resilience", func() {
```

Comment:
I think it's better to put ref_id here, don't need to have the same ref_id in each group of tests

Reply:
OK, done!